code/game/machinery/spaceheater.dm

```diff
@@ -203,7 +182,8 @@
 			power_change()
 		update_icon()
 
-/obj/machinery/space_heater/stationary//For mounting on walls in planetary buildings and stuff.
+//For mounting on walls in planetary buildings and stuff.
+/obj/machinery/space_heater/stationary
 	name = "stationary air conditioning unit"
 	desc = "A stationary air conditioning unit. It can heat or cool a room to your liking."
```

Comment:
```suggestion
	name = "stationary temperature control unit"
	desc = "A stationary temperature control unit. It can heat or cool a room to your liking."
```